stimela/backends/flavours/python_flavours.py

```diff
@@ -119,23 +119,33 @@ def get_arguments(self, cab: Cab, params: Dict[str, Any], subst: Dict[str, Any],
                         ).decode('ascii')
 
         # form up command string
+        if stimela.VERBOSE:
+            msg1 = f"""print("## importing {py_module}.{py_function}")"""
+            msg2 = f"""print(f"## invoking callable {command}({{repr(_inputs)}}) (as click command) using external interpreter")"""
+            msg3 = f"""print(f"## invoking callable {command}({{repr(_inputs)}}) using external interpreter")"""
+            msg4 = f"""print("## return value is ", _result)"""
+        else:
+            msg1 = msg2 = msg3 = msg4 = ""
         code = f"""
 import sys, json, zlib, base64
 _inputs = json.loads(zlib.decompress(
                         base64.b64decode(sys.argv[1].encode("ascii"))
                     ).decode("ascii"))
 sys.path.append('.')
+{msg1}
 from {py_module} import {py_function}
 try:
     from click import Command
 except ImportError:
     Command = None
 if Command is not None and isinstance({py_function}, Command):
-    print("invoking callable {command}() (as click command) using external interpreter")
+    {msg2}
     {py_function} = {py_function}.callback
 else:
-    print("invoking callable {command}() using external interpreter")
+    {msg3}
+    pass
```

Comment:
```suggestion
```

The `pass` after `print(...)` in the else clause is unnecessary.

Reply:
No, it's intentional. ``msg3`` can be an empty string, in this case the ``pass`` is needed or the interpreter complains of indentation error.

Reply:
Yes, that's correct. NVM.